### PR TITLE
Fix/deploy

### PR DESCRIPTION
**Pull request by Aurelsicoko**

### CODE REVIEW

File: docs/3.x.x/en/advanced/customize-admin.md [hunk 1]
```diff
@@ -111,90 +111,119 @@ This will replace the folder's content located at `./admin/admin/build`. Visit h
...
   "admin": {
     "build": {
-      "host": "https://admin.myapp.com",
-      "backend": "https://api.myapp.com:8080",
+      "host": "https://yourfrontend.com/dashboard", // Note: The custom path has moved directly in the host URL.
+      "backend": "https://yourbackend.com",
       "plugins": {
-        "source":  "host",
-        "folder": "plugins"
+        "source":  "host" // What does it means? The script tags in the index.html will use the host value to load the plugins (ex: https://yourfrontend.com/dashboard/plugins/content-manager/main.js).
```
Comment (edimitchel) [inline docs/3.x.x/en/advanced/customize-admin.md]: Missing coma after `"source":  "host"` ..

Comment (edimitchel) [reply docs/3.x.x/en/advanced/customize-admin.md]: !! Warning !!
JSON format doesn't accept comment ! 
Did you ever try start once?

Comment (edimitchel) [reply docs/3.x.x/en/advanced/customize-admin.md]: Sorry.. I miss that it's docs.. 👍

Comment (Aurelsicoko) [reply docs/3.x.x/en/advanced/customize-admin.md]: @edimitchel Thank you! It's fixed 👍

File: docs/3.x.x/en/advanced/customize-admin.md [hunk 2]
```diff
@@ -111,90 +111,120 @@ This will replace the folder's content located at `./admin/admin/build`. Visit h
...
+  "cron": {
+    "enabled": false
+  },
   "admin": {
+    "path": "/dashboard",
     "build": {
-      "host": "https://admin.myapp.com",
-      "backend": "https://api.myapp.com:8080",
+      "host": "/",  // Note: The administration will be accessible from the root of the domain (ex: https//yourfrontend.com/)
+      "backend": "http://yourbackend.com",
```
Comment (lauriejim) [inline docs/3.x.x/en/advanced/customize-admin.md]: It's `https`

**Pull request merged**